css/public_analytics.scss

```diff
@@ -74,24 +74,19 @@ form {
 section {
   // background: $white;
   // padding: 1em;
-  #current_visitors {
-    font-size: 8em;
-    margin: 0;
-    @include media($small-screen) {
-      font-size: 6em;
-    }
-  }
-  h3 {
-    margin: 10px 0;
-  }
+  // #current_visitors {
+  //   font-size: 8em;
+  //   margin: 0;
+  //   @include media($small-screen) {
+  //     font-size: 6em;
+  //   }
+  // }
+  // h3 {
+  //   margin: 10px 0;
+  // }
```

Comment:
comment

Reply:
@ryanwoldatwork I've removed all un-necessary comments from the styles.